problems/secondSmallest.js

```diff
@@ -6,8 +6,20 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+function secondSmallest(arr) {
+  let smallNum = Infinity;
+  let secSmallNum = Infinity;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] < smallNum) {
+      smallNum = arr[i];
+    }
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
This only needs one loop. If you give the if statement an else if, you can check if arr[i] is smaller than secSmallNum